logging/dedicated-cluster-logging.adoc

```diff
@@ -20,9 +19,11 @@ any missing logging components, and adjusts the logging environment accordingly.
 
 The `ClusterLogging` CR is based on the `ClusterLogging` custom resource
 definition (CRD), which defines a complete OpenShift Logging environment and
-includes all the components of the logging stack to collect, store and visualize
+includes all the components of the logging stack to collect, store, and visualize
 logs.
 
+The `retentionPolicy` parameter in the `ClusterLogging` custom resource (CR) defines how long the internal Elasticsearch log store retains logs.
```

Comment:
@sfortner-RH Do you want to review these changes to the logging topics in Dedicated?